guide/getting-started/technology-primer.md

```diff
@@ -73,264 +79,265 @@ It's both. Lowercase bitcoin refers to the currency. So you may have 1 bitcoin i
+Transactions get verified and stored by the entire network of nodes and miners who must constantly verify, exchange, and secure transactions they receive from wallets and other services. 
+
+While creating a transaction to move your funds from one address to another is common, setting up more complex spending rules is also possible. Such rules could require multiple parties to agree before moving funds, how much each is entitled to, and even the amount of time that must pass before a transfer. These transaction features make Bitcoin customizable so that it's possible to build other applications and even networks on top of the base layer.
+
+There are multiple scaling solutions built on top of Bitcoin's base layer, but we will be focusing on the Lightning Network in this guide.
 
 **More info**
-- [Explorer software]({{ '/guide/getting-started/software/#explorers' | relative_url }})
+- [Transaction lifecycle](/payments/transactions)
+
+<!--
+   While the base layer needs to have high integrity of decentralization, high security guarantees, and solid monetary principals...
+-->
 
 </div>
 
-## How are transactions processed?
+## What is a payment channel?
 
 <div class="center" markdown="1">
 
 {% include image.html
-   image = "/assets/images/guide/getting-started/technology/mempool.jpg"
-   retina = "/assets/images/guide/getting-started/technology/mempool@2x.jpg"
-   alt-text = "A user wallet pushing a transaction into the mempool"
+   image = "/assets/images/guide/getting-started/technology/payment-channel.jpg"
+   retina = "/assets/images/guide/getting-started/technology/payment-channel@2x.jpg"
+   alt-text = "Several payment channels settling on the Bitcoin blockchain"
    width = 400
    height = 400
    layout = "float-right-desktop"
 %}
 
-Every transaction needs to be confirmed before the recipient can consider the involved bitcoin theirs. This waiting line for new transactions is called the mempool. The Bitcoin network can only process a certain amount of transactions per day, so it may take longer to process your transactions during busy times. Transactions that involve higher fees are typically processed faster.
+A payment channel is a [joint account]({{'/guide/glossary/#multi-signature-wallet-multisig' | relative_url }}) where two parties agree to bypass recording their payments on the blockchain. Opening a payment channel involves locking up some funds together in a transaction which enables faster and cheaper subsequent payments. Instead of publishing a transaction for new payments, the two parties update their balance in the payment channel by creating a transaction but don't publish it.
 
 </div>
 
-## How are transactions confirmed?
+## The Bitcoin lightning payments network
```

Comment:
```suggestion
## The Lightning payments network
```
Having the extra word Bitcoin in there seems weird. And are we capitalizing "Lightning network" or not?